app/javascript/shopify_app/shopify_app.js

```diff
@@ -17,10 +17,8 @@ document.addEventListener("DOMContentLoaded", async () => {
 
   // Wait for the token before trying to load an authenticated page
   await retrieveToken(app);
-  Turbolinks.visit(data.loadPath);
```

Comment:
I think this is important.

Developers shouldn't display "logged in" content until they know what shop/user they're interacting with. Without this example how will they know how to orchestrate that?

Also, this model breaks if someone clicks the link to the products page before the token has been retrieved. They'll get a 4xx via AJAX instead of what they expect.

I appreciate the goal of simplifying but I think this has simplified too much.

Reply:
I've reintroduced this step of the pattern under the `SplashPageController`